ui/StatusQ/src/StatusQ/Components/StatusCard.qml

```diff
@@ -87,6 +92,16 @@ Rectangle {
        Used to set Tertiary text in the StatusCard
     */
     property alias tertiaryText: tertiaryText.text
+    /*!
+       \qmlproperty alias StatusCard::disableText
+       Used to set disable text in the StatusCard
```

Comment:
Not sure this best clarifies the ambiguity between `disabledText` and `disableText`. How about something on the line "Shown on hovered to indicate that click action will disable the card"?